doc/couchdbkit.org/buildweb.py

```diff
@@ -154,16 +155,16 @@ def generate_rss(self):
         rss.write_xml(open(os.path.join(conf.OUTPUT_PATH, "feed.xml"), "w"))
         
     def generate_sitemap(self):
-        xml = u'<?xml version="1.0" encoding="UTF-8"?>'
-        xml += u'<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">'
+        xml = '<?xml version="1.0" encoding="UTF-8"?>'
+        xml += '<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">'
         for page in self.sitemap:
-            xml += u'<url>'
-            xml += u'<loc>%s</loc>' % page.url
-            xml += u'<lastmod>%s</lastmod>' % rfc3339_date(page.headers['published'])
-            xml += u'<changefreq>daily</changefreq>'
-            xml += u'<priority>0.5</priority>'
-            xml += u'</url>'
-        xml += u'</urlset>'
+            xml += '<url>'
+            xml += '<loc>%s</loc>' % page.url
+            xml += '<lastmod>%s</lastmod>' % rfc3339_date(page.headers['published'])
+            xml += '<changefreq>daily</changefreq>'
+            xml += '<priority>0.5</priority>'
+            xml += '</url>'
+        xml += '</urlset>'
```

Comment:
wat

Reply:
Yeah. we should probably clean up a lot of the random crap in this repo. The upstream project is no longer maintained and I think we long gave up on PRing our jsonobject fork